### ISSUE TITLE
error: a run-time error occured - std::vector of char

### ISSUE THREAD

**Issue by gh2375** [NASA-SW-VnV/ikos#28]
ikos encounters a run-time error when `std::vectors` with a `char` as element type is used. This error does not occur when an `unsigned char` or `signed char` is used. Possibly related to https://github.com/NASA-SW-VnV/ikos/issues/20#issuecomment-447025699 and https://github.com/NASA-SW-VnV/ikos/issues/21#issue-390416639 but here provided with minimal working example.

### Minimal working example:
```
#include <vector>

int main(int argc, char * argv[])
{
    std::vector<char> v;

    return 0;
}
```

### Output:
```
$ ./ikos /home/username/test-vector.cpp -v
[*] Compiling C:/msys64/home/username/test-vector.cpp
[.] Running C:/msys64/mingw64/bin/clang.exe -c -emit-llvm -Wall -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0 -g -O0 -Xclang -disable-O0-optnone 'C:/msys64/home/username/test-vector.cpp' -o 'C:/Users/username/AppData/Local/Temp/ikos-3sw6dqmp/test-vector.bc' -isystem 'C:/msys64/home/username/ikos-2.1/install/include' -fcolor-diagnostics -std=c++14
[*] Running ikos preprocessor
[.] Running 'C:/msys64/home/username/ikos-2.1/install/bin/ikos-pp.exe' -opt=basic -entry-points=main 'C:/Users/username/AppData/Local/Temp/ikos-3sw6dqmp/test-vector.bc' -o 'C:/Users/username/AppData/Local/Temp/ikos-3sw6dqmp/test-vector.pp.bc'
[*] Running ikos analyzer
[.] Running 'C:/msys64/home/username/ikos-2.1/install/bin/ikos-analyzer.exe' -a=prover,sio,sound,fca,dfa,pcmp,shc,dca,nullity,boa,dbz -d=interval -entry-points=main -globals-init=skip-big-arrays -prec=mem -proc=inter -allow-dbg-mismatch -display-checks=no -display-inv=no -color=auto -log=debug 'C:/Users/username/AppData/Local/Temp/ikos-3sw6dqmp/test-vector.pp.bc' -o output.db
[.] Creating output database output.db
[.] Loading LLVM bitcode
[.] Verifying integrity of LLVM bitcode
[.] Checking for debug information
[*] Translating LLVM bitcode to AR
ikos-analyzer.exe: C:/Users/username/AppData/Local/Temp/ikos-3sw6dqmp/test-vector.pp.bc: error: unexpected null pointer in llvm::DICompositeType with DW_TAG_structure_type or DW_TAG_class_type tag
ikos: error: a run-time error occured
```

### System:
MSYS2

### Version:
```
$ ./ikos --version
ikos 2.1
```

**Comment by gh2375** [comment]
Confirmed fixed as of commit de1c7c0f698f56d8722c9f2cf51dbc7e6a1b3f03.

**Comment by arthaud** [comment]
Thank you.
